### PR TITLE
[WJ-254] Branding proposal: EstrellaYoshte

### PR DESCRIPTION
**Pull request by rossjrw**
[WJ-254](https://scuttle.atlassian.net/browse/WJ-254)

This PR replaces our current working logo as of #59 with one created by Wikidot user EstrellaYoshte (SYwaves), who has agreed to release it under CC-BY-SA 3.0.

<p align="center"></p>

<details><summary>Older versions of this logo</summary>
<p align="center"></p>
<p align="center"></p>
<p align="center"></p>
<p align="center"></p>
</details>

Please note that we are current amassing as many different branding options as possible and that this logo is only a candidate for our final logo.

This PR is to act as a marker for the branch that contains our copies of this logo draft in the repository. It is not to be merged unless we have decided that this logo will be the one that represent Wikijump, and until we have made an SVG version and tweaked it until everyone is happy with it. It should remain in draft until then.

Colors:
* Primary: Dark Navy Blue `#0068b5`
* Accent: Light Blue `#19a9ff`

Font: Poppins

Thank you again to EstrellaYoshte for designing this logo.

### CODE REVIEW

**Comment by emmiegit** [comment]
Since this is a release candidate, instead of removing the old logo, could we just rename it?

**Comment by rossjrw** [comment]
@ammongit Why's that? Is there any need to keep old logo versions in the repository?

**Comment by emmiegit** [comment]
@rossjrw well you were saying we'd have multiple versions until we've decided on for sure, at which point we can remove the others

File: assets/logo.inkscape.svg [hunk 1]
```diff
@@ -1,50 +1,131 @@
...
    xmlns:dc="http://purl.org/dc/elements/1.1/"
    xmlns:cc="http://creativecommons.org/ns#"
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:svg="http://www.w3.org/2000/svg"
    xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   sodipodi:docname="wikijump2.svg"
+   sodipodi:docname="logo.inkscape.svg"
```
Comment (emmiegit) [inline assets/logo.inkscape.svg:12]: Minor: rename to `wikijump.svg`?

File: assets/logo.inkscape.svg [hunk 2]
```diff
@@ -1,50 +1,131 @@
...
-   inkscape:export-filename="C:\Users\Ross\Documents\wikijump2.png"
    id="svg8"
    version="1.1"
-   viewBox="0 0 58.264759 16.036171"
-   height="16.036171mm"
-   width="58.264759mm">
+   viewBox="0 0 569.79474 141.29433"
+   height="141.29433mm"
+   width="569.79474mm"
+   inkscape:export-filename="C:\Users\Ross\Downloads\wikijump.png"
```
Comment (emmiegit) [inline assets/logo.inkscape.svg]: This meta field should be removed.

**Review by emmiegit — APPROVED** [review]

File: assets/logo.inkscape.svg (hunk 2 above)
Comment (rossjrw) [reply assets/logo.inkscape.svg]: Good point - there's probably a lot of metadata to strip. I wonder if there's a way to stop Inkscape from adding it?

Not much point removing it until we're on the precipice of merging, but definitely good to keep in mind for then

File: assets/logo-round.inkscape.svg [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+   xmlns:osb="http://www.openswatchbook.org/uri/2009/osb"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   inkscape:export-filename="C:\Users\Ross\Downloads\logo.inkscape.png"
```
Comment (emmiegit) [inline assets/logo-round.inkscape.svg]: Like the other files, can we remove this?

File: assets/logo-square.inkscape.svg [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+   xmlns:osb="http://www.openswatchbook.org/uri/2009/osb"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   inkscape:export-filename="C:\Users\Ross\Downloads\logo.inkscape.png"
```
Comment (emmiegit) [inline assets/logo-square.inkscape.svg]: Ditto

**Review by emmiegit — APPROVED** [review]

**Comment by emmiegit** [comment]
Per the discussion in #wikijump-branding, I think this is good to merge.

**Comment by emmiegit** [comment]
Going to merge this.

**Pull request merged**